build.sh

```diff
@@ -43,7 +43,7 @@ LAST_COMMIT_TIMESTAMP=$(git log -1 --format=%ct)
 major=1
 minor=0
 # no. of days since epoch
-build=$(($LAST_COMMIT_TIMESTAMP/3600/24))
+build=0
```

Comment:
Me @piotrpMSFT and @Sridhar-MS had an offline conversation to unify that and we decided to unify that so that a third number is 0 (we can't control easily that number for nupkgs). We are aware it will break some people but it is better to do it now when we don't have that many users than later.